mods/cnc/maps/nod06c/map.yaml

```diff
@@ -0,0 +1,634 @@
+	Actor68: hq
+		Location: 28,20
+		Owner: GDI
+	Actor69: gtwr
+		Location: 32,24
+		Owner: GDI
+	Actor72: jeep
+		Location: 37,40
+		Owner: GDI
+		Facing: 96
+	Actor73: jeep
+		Location: 47,18
+		Owner: GDI
+		Facing: 160
+	Actor74: mtnk
+		Location: 17,39
+		Owner: GDI
+	Actor75: mtnk
+		Location: 16,39
+		Owner: GDI
+	Actor76: mtnk
+		Location: 15,39
+		Owner: GDI
+	Actor83: mtnk
+		Location: 39,27
+		Owner: GDI
+		Facing: 96
+	Actor85: jeep
+		Location: 29,30
+		Owner: GDI
+		Facing: 224
+	Actor86: e3
+		Location: 35,36
+		Owner: GDI
+		SubCell: 0
+	Actor87: e1
+		Location: 35,36
+		Owner: GDI
+		SubCell: 4
+	Actor88: e1
+		Location: 35,36
+		Owner: GDI
+		SubCell: 3
+	Actor89: e1
+		Location: 49,22
+		Owner: GDI
+		Facing: 128
+		SubCell: 1
+	Actor92: e2
+		Location: 16,25
+		Owner: GDI
+		SubCell: 3
+	Actor93: e2
+		Location: 15,26
+		Owner: GDI
+		SubCell: 2
+	Actor96: e3
+		Location: 36,25
+		Owner: GDI
+		Facing: 96
+		SubCell: 2
+	Actor99: e1
+		Location: 30,26
+		Owner: GDI
+		Facing: 96
+		SubCell: 4
+	Actor100: e1
+		Location: 48,21
+		Owner: GDI
+		Facing: 160
+		SubCell: 4
+	Actor101: e3
+		Location: 51,42
+		Owner: GDI
+		Facing: 128
+		SubCell: 2
+	Actor102: e3
+		Location: 52,42
+		Owner: GDI
+		Facing: 128
+		SubCell: 2
+	Actor103: e1
+		Location: 40,30
+		Owner: GDI
+		Facing: 96
+		SubCell: 2
+	Actor104: e2
+		Location: 47,35
+		Owner: GDI
+		SubCell: 1
+	Actor105: e2
+		Location: 35,46
+		Owner: GDI
+		Facing: 128
+		SubCell: 1
+	Actor114: e2
+		Location: 43,29
+		Owner: GDI
+		SubCell: 4
+	Actor118: e3
+		Location: 41,29
+		Owner: GDI
+		SubCell: 0
+	Actor119: e2
+		Location: 27,31
+		Owner: GDI
+		SubCell: 1
+	Actor120: e2
+		Location: 23,24
+		Owner: GDI
+		Facing: 128
+		SubCell: 4
+	Actor121: e2
+		Location: 25,24
+		Owner: GDI
+		Facing: 128
+		SubCell: 3
+	Actor122: e2
+		Location: 27,31
+		Owner: GDI
+		SubCell: 2
+	waypoint27: waypoint
+		Location: 17,50
+		Owner: Neutral
+	waypoint26: waypoint
+		Location: 46,46
+		Owner: Neutral
+	waypoint25: waypoint
+		Location: 17,49
+		Owner: Neutral
+	waypoint10: waypoint
+		Location: 16,50
+		Owner: Neutral
+	waypoint7: waypoint
+		Location: 31,26
+		Owner: Neutral
+	waypoint6: waypoint
+		Location: 37,25
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 42,42
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 42,56
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 55,56
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 55,44
+		Owner: Neutral
+	waypoint1: waypoint
+		Location: 55,32
+		Owner: Neutral
+	waypoint0: waypoint
+		Location: 43,32
+		Owner: Neutral
+	UnitsEntryLeft: waypoint
+		Location: 56,57
+		Owner: Neutral
+	UnitsRallyLeft: waypoint
+		Location: 53,51
+		Owner: Neutral
+	UnitsEntryMiddle: waypoint
+		Location: 57,57
+		Owner: Neutral
+	UnitsRallyMiddle: waypoint
+		Location: 55,50
+		Owner: Neutral
+	UnitsEntryRight: waypoint
+		Location: 58,57
+		Owner: Neutral
+	UnitsRallyRight: waypoint
+		Location: 57,51
+		Owner: Neutral
+	ChnEnrty: waypoint
```

Comment:
ChnEntry. Funnily enough, you made the same typo in the lua script as well, so it all works :)